### ISSUE TITLE
Cannot read property 'duration' of undefined

### ISSUE THREAD

**Issue by timsim00** [choojs/choo#601]
### Expected behavior
No error.

### Actual behavior
logger.js:47 Uncaught TypeError: Cannot read property 'duration' of undefined
    at logger.js:47
    at index.js:79
    at index.js:33
    at index.js:23 

hooks.on('render', function (timings) {
    if (!timings) return logger('info')('render')
47> var duration = timings.render.duration.toFixed()

Getting this error on every page reload/refresh.

### Steps to reproduce behavior
Static (client only) app started this way:

const tree = app.start();
document.body.appendChild(mainView());
const mainWrapper = document.getElementById('approot');
mainWrapper.appendChild(tree);

function mainView (state, emit) {
  return html`
      <div id="approot"></div>
  `
}

Removing API call from DOMContentLoaded listener eliminates the error:    emitter.on('DOMContentLoaded', function () {...API call...})

Error did not occur when starting app this way:

if (!module.parent) {
  app.mount('body')
} else {
  module.exports = app
}

**Comment by YerkoPalma** [comment]
Had the same error before, but only related to choo-devtools (choo-log before)

see: https://github.com/choojs/choo-log/issues/84

**Comment by yoshuawuyts** [comment]
Heya, what's your OS / Browser versions? This is not good; I thought we solved this by adding `try…catch` everywhere, but there might be some latent browser bugs at play here. Thanks!

**Comment by YerkoPalma** [comment]
Mmm there are some weird things here. First, why are you trying it like this

```js
const tree = app.start();
document.body.appendChild(mainView());
const mainWrapper = document.getElementById('approot');
mainWrapper.appendChild(tree);
```

Second, as I can see, you are not using devtools module, where I had the issue before. And finally, what do you mean by 

> Removing API call from DOMContentLoaded listener eliminates the error: emitter.on('DOMContentLoaded', function () {...API call...})

How does that API call looks like?

**Comment by timsim00** [comment]
Here's [the app](https://github.com/timsim00/choo-baqend-starter)
Chrome Version 62.0.3202.94 (Official Build) (64-bit),  OSX 10.12.6: ...error does not occur on Firefox.

>First, why are you trying it like this
I'm not totally familiar with how to structure a choo app.  App was created with create-choo-app so I followed that pattern with each 'page' returning "html`<body></body`.  This caused a couple problems...1 If I used bel() more than once in a view it would say html was not a function.  2. Baqend is attaching an iframe to body so returning body would wipe out anything attached to it...plus I think we don't want to replace <body> on every view...and sometimes we want to attach other things to <body> when the app loads, like analytics.  I'm open to other ways to structure the app...there aren't a ton of examples to go by.  This one worked but introduced the issue we're discussing now.

>you are not using devtools module
I'm using devtools.  If I comment out "app.use(require('choo-devtools')())" the error goes away.

>And finally, what do you mean by
Commenting out [this line](https://github.com/timsim00/choo-baqend-starter/blob/master/stores/app.js#L30) also causes the issue to go away.
